modules/undertoneBidAdapter.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * Adapter to send bids to Undertone
+ */
+
+import * as utils from 'src/utils';
+import { registerBidder } from 'src/adapters/bidderFactory';
+
+const BIDDER_CODE = 'undertone';
+const URL = '//hb.undertone.com/hb'; // //ads.undertone.com/hb
+
+export const spec = {
+  code: BIDDER_CODE,
+  isBidRequestValid: function(bid) {
+    if (bid && bid.params && bid.params.publisherId && bid.params.placementId) {
+      bid.params.publisherId = parseInt(bid.params.publisherId);
+      return true;
+    }
+  },
+  buildRequests: function(validBidRequests) {
+    const payload = {
+      'x-ut-hb-params': []
+    };
+    const timeout = window.PREBID_TIMEOUT || null;
+    const host = utils.getTopWindowLocation().host;
+    const domain = /[-\w]+\.(?:[-\w]+\.xn--[-\w]+|[-\w]{3,}|[-\w]+\.[-\w]{2})$/i.exec(host);
+
+    const pubid = validBidRequests[0].params.publisherId;
+    const REQ_URL = `${URL}?pid=${pubid}&domain=${domain}`;
+
+    validBidRequests.map(bidReq => {
+      const bid = {
+        bidRequestId: bidReq.bidId,
+        hbadaptor: 'prebid',
+        domain: domain,
+        placementId: bidReq.params.placementId,
+        publisherId: bidReq.params.publisherId,
+        sizes: bidReq.sizes,
+        timeout: timeout,
+        params: bidReq.params
+      };
+      payload['x-ut-hb-params'].push(bid);
+    });
+    return {
+      method: 'POST',
+      url: REQ_URL,
+      withCredentials: true,
+      data: JSON.stringify(payload)
+    };
+  },
+  interpretResponse: function(serverResponse, request) {
+    const bids = [];
```

Comment:
#1748 changed the first argument of `interpretResponse` to:

```
{
  body: responseBody,
  headers: {
    get: function(header) { /* returns a header from the HTTP response */ }
  }
}
```

so adding something like

`serverResponse = serverResponse.body;`

just below this line, or however you'd prefer to grab the body, and updating corresponding tests if needed should get this back to working properly